{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -83,6 +83,15 @@
         # The node container isn't started (yet?)
         pass
     {%- endif %}
+{% if cookiecutter.windows == 'y' %}
```

Comment:
I'm not sure it will change much in the final formatting, but I tend to add a minus sign to [trim empty lines before the Jinja tags](https://jinja.palletsprojects.com/en/2.10.x/templates/#whitespace-control), and add an explicit emtpy line after:

```suggestion
{%- if cookiecutter.windows == 'y' %}

```

I found it easier to see where the empty line will be when the codition is true or false.

Reply:
Yeah, it looks better. and also it should be add indent too.